editor/export/project_export.cpp

```diff
@@ -1236,11 +1252,22 @@ ProjectExportDialog::ProjectExportDialog() {
 	name = memnew(LineEdit);
 	settings_vb->add_margin_child(TTR("Name:"), name);
 	name->connect("text_changed", callable_mp(this, &ProjectExportDialog::_name_changed));
+
 	runnable = memnew(CheckButton);
 	runnable->set_text(TTR("Runnable"));
 	runnable->set_tooltip_text(TTR("If checked, the preset will be available for use in one-click deploy.\nOnly one preset per platform may be marked as runnable."));
 	runnable->connect("pressed", callable_mp(this, &ProjectExportDialog::_runnable_pressed));
-	settings_vb->add_child(runnable);
+
+	advanced_options = memnew(CheckButton);
+	advanced_options->set_text(TTR("Advanced Options"));
+	advanced_options->set_tooltip_text(TTR("If checked, the advanced options will be shown"));
```

Comment:
Finish tooltip with a period for consistency with Runnable's tooltip:

```suggestion
	advanced_options->set_tooltip_text(TTR("If checked, the advanced options will be shown."));
```

Reply:
Fixed!